File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.java
class Solution {
    public int solution(String t, String p) {
        int answer = 0;
     
        for(int i=0; i<=t.length()- p.length(); i++){
            String num = t.substring(i,i+p.length());
            if(Long.parseLong(num) <= Long.parseLong(p)) {
                answer++;
            }
        }
        return answer;
    }
}

File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.js
function solution(t, p) {
    var answer = 0;
    
    for(var i=0; i<=t.length-p.length; i++){
        var strNum = ''; 
        
        for(var j=0; j<p.length; j++){
            strNum += t.charAt(i+j);
        }
        
        if(Number(strNum) <= Number(p)){
            answer++;
        }
    }
    return answer;
}